src/persist-client/src/lib.rs

```diff
@@ -569,6 +592,177 @@ mod tests {
         assert_eq!(listen.read_until(&6).await, expected_events);
     }
 
+    // Appends need to be contiguous for a shard, meaning the lower of an appended batch needs to
+    // match the current shard upper.
+    #[tokio::test]
+    async fn contiguous_append() {
+        mz_ore::test::init_logging_default("info");
```

Comment:
nit: info is the default, just use `mz_ore::test::init_logging()`, here and elsewhere

Reply:
> nit: info is the default, just use mz_ore::test::init_logging(), here and elsewhere
:ok_hand: